code/controllers/configuration/configuration.dm

```diff
@@ -432,6 +432,9 @@ Example config:
 				currentmap = null
 			if ("disabled")
 				currentmap = null
+			if("feedbacklink")
+				if(currentmap.map_name == SSmapping.current_map.map_name)
```

Comment:
why? isn't currentmap the same datum as SSmapping's thing? or is that not the case

Reply:
currentmap is just the map config it's currently looping through to load config, while the other is the one being played